test/corpus/cases.txt

```diff
@@ -218,3 +218,38 @@ case var {
             (discard)))
         (record
           (constructor_name))))))
+
+================================================================================
+Case with int remainder in guard
+================================================================================
+
+case var {
+  _ if 11 % 2 == 0 -> True
```

Comment:
what's the associativity of `%` - would it or `*`/`/` or `+`/`-` take precedence for example in something like `11 + 1 % 2`?

Reply:
I believe in other languages like Rust it has the same precedence as multiplication and division so I think it's correct at `6`. We can always revisit this later if expressions are sticking together incorrectly